static/styles/zulip.css

```diff
@@ -458,6 +458,13 @@ li,
     margin-top: -24px;
 }
 
+.copy_invite_link {
```

Comment:
You could just use the id here (`#copy_generated_invite_link`), instead of defining another class and using it - and thus avoid the other class.